src/vm/comcallablewrapper.cpp

```diff
@@ -958,18 +958,24 @@ void SimpleComCallWrapper::BuildRefCountLogMessage(LPCWSTR wszOperation, StackSS
 
         if (ETW_EVENT_ENABLED(MICROSOFT_WINDOWS_DOTNETRUNTIME_PRIVATE_PROVIDER_Context, CCWRefCountChange)) 
         {
-            SString className;
-            className.SetUTF8(pszClassName);
-            SString nameSpace;
-            nameSpace.SetUTF8(pszNamespace);
-
-            FireEtwCCWRefCountChange(
-                handle, 
-                (Object *)obj, 
-                this, 
-                dwEstimatedRefCount, 
-                NULL,                   // domain value is not interesting in CoreCLR
-                className.GetUnicode(), nameSpace.GetUnicode(), wszOperation, GetClrInstanceId());
+            EX_TRY
```

Comment:
Has something changed that now allows the functions in here to throw? Has this function already thrown, just never had a catch around it?

Reply:
This was caused by a previous change of mine: https://github.com/dotnet/coreclr/pull/17989.  The uses of SString can throw.